src/analyzer/Tag.h

```diff
@@ -43,7 +43,7 @@ class Tag : public ::Tag  {
 	 * Returns false if the tag represents an error value rather than a
 	 * legal analyzer type.
 	 */
-	explicit operator bool() const	{ return *this != Tag(); }
+	explicit operator bool() const	{ return *this != Error; }
```

Comment:
Is there any concern that `Error` isn't `const` and could technically be modified?

Reply:
Yeah, concern in the sense a mistake can slip in, but not in the sense that we ever want intentional modifications, so may as well make them `const` now.  Thanks for flagging.